instrumentation/opentelemetry-instrumentation-system-metrics/src/opentelemetry/instrumentation/system_metrics/__init__.py

```diff
@@ -164,12 +164,12 @@ def __init__(
 
         self._system_thread_count_labels = self._labels.copy()
 
-        self._runtime_memory_labels = self._labels.copy()
-        self._runtime_cpu_time_labels = self._labels.copy()
-        self._runtime_gc_count_labels = self._labels.copy()
-        self._runtime_thread_count_labels = self._labels.copy()
-        self._runtime_cpu_utilization_labels = self._labels.copy()
-        self._runtime_context_switches_labels = self._labels.copy()
+        self._process_memory_labels = self._labels.copy()
+        self._process_cpu_time_labels = self._labels.copy()
+        self._process_gc_count_labels = self._labels.copy()
```

Comment:
I was going to say this looked more process.runtime but I don't see this is defined at all in the semantic conventions